05.android/02.installation/docs.en.md

```diff
@@ -37,21 +37,46 @@ You will see the request to save the downloaded file. Press **OK**.
 
 When the download is complete, the system will ask you if you want to install the AdGuard app. Press **Install**.
 
-![](android_install_EN_04.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/InstallMenuEn.png" />
 
-In Android 4, after AdGuard has been downloaded, you may see a dialogue window requesting for further action regarding the file. Select **Package installer** and then **Just once**.
+Wait for the installation procedure to finish. When it is finished, press the **Open** button. If you decide to postpone the launch of the application for another time, press **Done**.
 
-![](android_install_EN_05.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/Installmenu2En.png" />
 
-If you use Android 4, next you will see the list of permissions required by the app. Press **Next** and then **Install**.
+You will then be asked to read the **License agreement** and AdGuard's privacy policy. 
+You can participate in product development. 
+To do this, you should check the box *"Allow AdGuard to send automatic error reports"*.
+After that, you can pre-configure AdGuard protection by clicking **Continue**, or you can do it later by clicking **Skip quick settings**.
 
-![](android_install_EN_06.png?cropResize=400,600)
+<img src="https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/Android3.1/installmainEn.png" />
 
-Wait for the installation to complete and press **Done**.
+When you select the first option, a dialog box opens where you have to choose the level of privacy protection.
```

Comment:
If you select the first option, a new dialog box will appear where you can configure privacy protection settings.